packages/forklift-console-plugin/src/modules/Providers/utils/helpers/safeBase64Decode.ts

```diff
@@ -1,9 +1,13 @@
 import { Base64 } from 'js-base64';
 
+/**
+ * @returns - in case of a decoding failure returns 'undefined'
+ * (if value is set to undefined) or an empty string otherwise.
+ */
 export function safeBase64Decode(value: string) {
   try {
     return Base64.decode(value);
   } catch {
-    return '';
+    return value !== undefined ? '' : undefined;
```

Comment:
The previous implementation returned  `''` in case of any failure. An `undefined` input is one of those cases.

As mentioned in the  func's comment, I changed it to return:
-  `undefined` only if the input is `undefined` (i.e., this field supports `undefined` as a legitimate and a separate status).
- and  `''` for all other cases, keeping the previous behavior.

Not sure that replacing it to return `undefined` for all usages is safe. 
We can go over the code in a separate PR and verify that. I started doing that and saw that it's not surely is the case.

Reply:
> Not sure that replacing it to return undefined for all usages is safe.

lets do that now, if it's not safe to return `undefined` for some error, it may not be safe to return `undefined` if it's `undefined`

I hope we don't use this method in lots of places, check how it is used, if it is safe, return `undefined` for all errors

Reply:
@yaacov  As I wrote above,  I checked that already when changing this function and it is used in few places where `undefined` is not expected. 
Since it is a separate wide fix, better saving current backward behavior and fixing that in a separate PR.

Reply:
my concern here is that we have places where we can't handle 'undefined' while using a method that may return this value, what do you suggest ?

Reply:
With current code it's safe since we pass the `undefined`  only when we can handle it correctly. This is the current fix. For all other cases, we return an `empty string` as was the prefix behavior.

On a follow up PR, we'll fix the code such that `undefined` will be safe for all current `safeBase64Decode` usages.